app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,95 @@
+<div class="signup-container">
+  <h2>Sign up</h2>
+
+  <form action="<%= registration_path(resource_name) %>" method="post">
+    <%= csrf_meta_tags %>
+
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="form-group">
+      <label for="user_name">Name</label>
+      <input type="text" id="user_name" name="user[name]" placeholder="Enter your name" required>
+    </div>
+
+    <div class="form-group">
+      <label for="user_email">Email</label>
+      <input type="email" id="user_email" name="user[email]" placeholder="Enter your email" required>
+    </div>
+
+    <div class="form-group">
+      <label for="user_password">Password</label>
+      <input type="password" id="user_password" name="user[password]" placeholder="Enter your password" required>
+    </div>
+
+    <div class="form-group">
+      <label for="user_password_confirmation">Confirm Password</label>
+      <input type="password" id="user_password_confirmation" name="user[password_confirmation]" placeholder="Confirm your password" required>
+    </div>
+
+    <div class="form-group">
+      <input type="submit" value="Sign up">
+    </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you also add a `textarea` field for `Bio` input,... in the `Sign Up` form to help a user to be registered with all needed information